android/java/org/chromium/chrome/browser/crypto_wallet/util/ParsedTransaction.java

```diff
@@ -568,7 +568,8 @@ public Pair<String, String> makeTxListItemTitles(Context context) {
                     + "0x Exchange Proxy";
```

Comment:
nit: shouldn't "Exchange Proxy" be translated via resources? maybe later

Reply:
good question, I think we did it without any translation initially as it's like a name of a service, like `Brave` or something, but maybe we need to, cc @jamesmudgett do we translate it?